test/runtests.jl

```diff
@@ -31,4 +31,8 @@ end
   include("compiler.jl")
 end
 
+@testset "CUDA tests" begin
```

Comment:
You'll need to check that CUDA is available before running these, as in the Flux tests. It would be nice to also have a similar `@info` message to show whether we are doing GPU testing or not.

Reply:
Right, I've added that messaging. Wondering if I should use CUDAapi like we do with Flux